frontend/src/charts/choroplethMap/tooltipUtils.ts

```diff
@@ -35,7 +35,7 @@ export const formatMetricValue = (
   metricConfig: MetricConfig,
   isLegendLabel?: boolean,
 ): string => {
-  if (value === undefined) return 'no data'
+  if (value === undefined || value === null) return 'no data'
```

Comment:
nice! this is the most explicit way and I think we should keep it like this. just FYI in case you see it elsewhere, sometimes people write `value == null` and that means the same thing, the _double_ equal loosely will evaluate for undefined or null